include/cantera/oneD/StFlow.h

```diff
@@ -325,25 +312,144 @@ class StFlow : public Domain1D
+     * the data in the qdotRadiation variable.
+     *
+     * The simple radiation model used was established by Liu and Rogg
+     * @cite liu1991. This model considers the radiation of CO2 and H2O.
+     *
+     * This model uses the optically thin limit and the gray-gas approximation to
+     * simply calculate a volume specified heat flux out of the Planck absorption
+     * coefficients, the boundary emissivities and the temperature. Polynomial lines
+     * calculate the species Planck coefficients for H2O and CO2. The data for the
+     * lines are taken from the RADCAL program @cite RADCAL.
+     * The coefficients for the polynomials are taken from
+     * [TNF Workshop](https://tnfworkshop.org/radiation/) material.
+     */
+    void computeRadiation(double* x, size_t jmin, size_t jmax);
+
+    /**
+     * Evaluate the continuity equation residual.
+     *
+     * This function calculates the residual of the continuity equation
+     * @f[
+     *     \frac{d(\rho u)}{dz} + 2\rho V = 0
+     * @f]
+     *
+     * Axisymmetric flame:
+     *  The continuity equation propagates information from right-to-left.
+     *  The @f$ \rho u @f$ at point 0 is dependent on @f$ \rho u @f$ at point 1 but not
+     *  on @f$ \dot{m} @f$ from the inlet.
+     *
+     * Freely-propagating flame:
+     *  The continuity equation propagates information away from a fixed temperature
+     *  point that is set in the domain.
+     *
+     * Unstrained flame: specified mass flux; the main example being
+     *   burner-stabilized flames.
+     *
+     * The default boundary condition for the continuity equation is zero velocity
+     * (@f$ u @f$) at the left and right boundary.
+     */
+    virtual void evalContinuity(double* x, double* rsd, int* diag,
+                                double rdt, size_t jmin, size_t jmax);
+
+    /**
+     * Evaluate the momentum equation residual.
+     *
+     * The function calculates the radial momentum equation defined as
+     * @f[
+     *    \rho u \frac{dV}{dz} + \rho V^2 =
+     *    \frac{d(\mu \frac{dV}{dz})}{dz} - \Lambda
+     * @f]
+     *
+     * The radial momentum equation is used for axisymmetric flows, and incorporates
+     * terms for time and spatial variations of radial velocity (@f$ V @f$). The
+     * default boundary condition is zero radial velocity (@f$ V @f$) at the left
+     * and right boundary.
+     */
+    virtual void evalMomentum(double* x, double* rsd, int* diag,
                               double rdt, size_t jmin, size_t jmax);
 
+    /**
+     * Evaluate the lambda equation residual.
+     *
+     * The function calculates the lambda equation as
+     * @f[
+     *    \frac{d\Lambda}{dz} = 0
+     * @f]
+     *
+     * The lambda equation serves as an eigenvalue that allows the momentum
+     * equation and continuity equations to be simultaneously satisfied in
+     * axisymmetric flows. The lambda equation propgates information from
```

Comment:
```suggestion
     * axisymmetric flows. The lambda equation propagates information from
```
Here and a couple other places.